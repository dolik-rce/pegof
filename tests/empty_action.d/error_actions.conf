input empty_action.d/error_actions.peg
optimize empty-action
header never
