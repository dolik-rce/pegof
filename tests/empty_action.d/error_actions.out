x <- "a" "b" "c" ~ { {} } "d" "e"
